Stripe/StripeiOSTests/LinkInlineSignupElementSnapshotTests.swift

```diff
@@ -124,6 +126,7 @@ extension LinkInlineSignupElementSnapshotTests {
 
     func makeSUT(
         saveCheckboxChecked: Bool = false,
+        prefilledEmailAddress: String? = nil,
         emailAddress: String? = nil,
```

Comment:
I'm having trouble understanding the difference between `emailAddress` and `prefilledEmailAddress` -- maybe it would help to update the name of `emailAddress` to differentiate between the two, or maybe a comment on what each parameter is instrumenting?

Reply:
Will rename them for clarity